data/101/911/123/101911123.geojson

```diff
@@ -117,15 +71,10 @@
         }
     ],
     "wof:id":101911123,
-    "wof:lang":[
-        "spa"
```

Comment:
Should this be kept, but moved to Norwegian?

Reply:
Added `lang_x_spoken` and `lang_x_official` in most recent commit.